avocado/core/data_dir.py

```diff
@@ -212,7 +220,7 @@ def __del__(self):
 _tmp_tracker = _TmpDirTracker()
 
 
-def get_tmp_dir():
+def get_tmp_dir(basedir=None, keep_tmp=False):
```

Comment:
Here I'd use `None` for `keep_tmp` as the user's not specifying it. I think it's better to leave it on the implementation directly inside the function, than specifying the default here. (not a strong opinion, though)